### PR TITLE
Fixed typos, grammar and punctuation in French translations.

### PR DESCRIPTION
**Pull request by Tititesouris**

### CODE REVIEW

File: app/src/main/res/values-fr-rFR/strings.xml [hunk 1]
```diff
@@ -4,38 +4,38 @@
     <string name="app_name">Note Crypt Pro</string>
     <string name="action_changePassword">Changer le mot de passe</string>
     <string name="action_newNote">Ajouter une note</string>
-    <string name="action_about">A propos</string>
+    <string name="action_about">À propos</string>
     <string name="hint_field_new_password">Nouveau mot de passe</string>
     <string name="hint_field_conf_password">Confirmer le mot de passe</string>
     <string name="title_changeMasterPass">Saisir un mot de passe principal</string>
-    <string name="title_databaseLocation">Nom de la base de données:</string>
+    <string name="title_databaseLocation">Nom de la base de données :</string>
```
Comment (RyuzakiKK) [inline app/src/main/res/values-fr-rFR/strings.xml:11]: I think there you should keep the colon without a space, unless it's a french convention that I'm not aware to.

**Review by RyuzakiKK — CHANGES_REQUESTED** [review]
Thank you!
Check the comment I left. Probably you just need to remove the space before the colons in the 4 lines where you added it.

File: app/src/main/res/values-fr-rFR/strings.xml (hunk 1 above)
Comment (Tititesouris) [reply app/src/main/res/values-fr-rFR/strings.xml:11]: No this is actually how punctuation works in French. !, ? : and ; all have a space before and after.

Comment (RyuzakiKK) [reply app/src/main/res/values-fr-rFR/strings.xml:11]: Good to know. Thanks.

**Review by RyuzakiKK — APPROVED** [review]

**Pull request merged**